### ISSUE TITLE
Windows console not drawing scroll arrows

### ISSUE THREAD

**Issue by d-ank** [mum4k/termdash#245]
https://github.com/mum4k/termdash/blob/fe3e973531917978dd6d412520528bf10e092087/widgets/text/text.go#L128
https://github.com/mum4k/termdash/blob/fe3e973531917978dd6d412520528bf10e092087/widgets/text/text.go#L147
[These characters are unable to be drawn in the windows console, or at least in the current rendition.](https://i.imgur.com/FwhQTdM.png)
maybe give an option to the text widget to swap the scroll up and down arrows out for ▲ and ▼ (or ↑ and ↓)?

**Comment by mum4k** [comment]
Hi @d-ank, thank you for the suggestion. I agree that this is a good solution and should be fairly trivial to implement.

Is this something you would be willing to contribute?

**Comment by Amirhossein2000** [comment]
Hi
```Go
cells, err := cvs.SetCell(cur, '⇧') --> cells, err := cvs.SetCell(cur, ' ▲')
```
Is this as easy as this or I'm wrong?

**Comment by d-ank** [comment]
Sorry for the delayed response, I will put in a pr in when I am at my desktop.

**Comment by mum4k** [comment]
@Amirhossein2000 that is the essence of the code change, yes. However if done like this, it would change the default behavior for all the users. What might be better is for us to add an option that would allow the user to specify the character to use in place of the arrows. One example of implementing such option is in the Gauge widget that allows setting a custom Gauge character:

https://github.com/mum4k/termdash/blob/fe3e973531917978dd6d412520528bf10e092087/widgets/gauge/options.go#L83-L89

@d-ank no pressure, take your time and thank you for looking into this.

**Comment by d-ank** [comment]
https://github.com/mum4k/termdash/pull/248 Here is the accompanying fix and addition.

**Comment by mum4k** [comment]
Released in #266.
